src/app/ui/portfolio/PortfolioHeaderBreakdown.tsx

```diff
@@ -63,6 +70,35 @@ export const PortfolioHeaderBreakdown = () => {
     return formattedPositions;
   }, [formattedPositions, portfolioWelcomeScreenClosed]);
 
+  useEffect(() => {
+    if (
+      isLoading ||
+      portfolioHasBeenTracked.current ||
+      !portfolioWelcomeScreenClosed
+    ) {
+      return;
+    }
+    portfolioHasBeenTracked.current = true;
```

Comment:
move that line after the track event to give it a chance to retry in case of error?